addons/interact_menu/XEH_clientInit.sqf

```diff
@@ -104,3 +104,25 @@ addMissionEventHandler ["Draw3D", DFUNC(render)];
         }];
     };
 }] call EFUNC(common,addEventHandler);
+
+
+//Debug to help end users identify mods that break CBA's XEH
+["SettingsInitialized", {
+    private ["_badClassnames"];
+    _badClassnames = [];
+    {
+        if ((isNil (format [QGVAR(Act_%1), typeOf _x])) || {isNil (format [QGVAR(SelfAct_%1), typeOf _x])}) then {
+            if (!((typeOf _x) in _badClassnames)) then {
+                _badClassnames pushBack (typeOf _x);
+                ACE_LOGERROR_3("Compile checks bad for (classname: %1)(addon: %2) %3", (typeOf _x), (unitAddons (typeOf _x)), _x);
+            };
+        };
+    } forEach (allUnits + vehicles);

```

Comment:
Extremly minor, but

`allUnits + allDeadMen + vehicles`

would include dead soldiers which are not covered by `allUnits` for god knows why.